plugins/GSdx/GSSetting.cpp

```diff
@@ -142,6 +140,18 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_UNSCALE_POINT_LINE:
 			return "Increases the width of lines at higher than native resolutions. This ensures that the lines will keep the correct proportions and prevents aliasing. "
 				"Avoids empty lines on the screen in games such as Ridge Racer V, and clears FMVs obscured by a grid in games like the Silent Hill series and Dirge of Cerberus.";
+		case IDC_MEMORY_WRAPPING:
+			return "Emulates GS memory wrapping accurately. This fixes issues where part of the image is cut-off by block shaped sections such as the FMVs in Wallace & Gromit: The Curse of the Were-Rabbit and Thrillville.\n"
+				"Note: This hack can have a small impact on performance.";
+		case IDC_MERGE_PP_SPRITE:
+			return "Replaces post-processing multiple paving sprites by a single fat sprite. It reduces various upscaling lines.\n"
+				"Note: This hack is a work in progress.";
+		case IDC_GEOMETRY_SHADER_OVERRIDE:
+			return "Allows the GPU instead of just the CPU to transform lines into sprites. This reduces CPU load and bandwidth requirement, but it is heavier on the GPU.\n"
+				"Automatic detection is recommended.\nNote: This option is only supported by GPU's which support at least DirectX10.";
```

Comment:
`GPUs`

And `Direct3D10` would probably be better (I remember there are people who will complain about this).